timesketch/frontend/src/views/Intelligence.vue

```diff
@@ -22,34 +22,161 @@ limitations under the License.
+
+            <b-table-column field="edit" label="" v-slot="props">
+              <span class="icon is-small" style="cursor:pointer;" title="Edit IOC" @click="startIOCEdit(props.row)"
+                ><i class="fas fa-edit"></i>
+              </span>
+            </b-table-column>
+
+            <b-table-column field="delete" label="" v-slot="props">
+              <span
+                class="icon is-small"
+                style="cursor:pointer;color:red"
+                title="Delete IOC"
+                @click="deleteIoc(props.row)"
+                ><i class="fas fa-trash"></i>
+              </span>
+            </b-table-column>
+          </b-table>
+          <!-- End IOC table, empty palceholder follows -->
+          <div v-else class="card-content">
+            Examine events in the <router-link :to="{ name: 'Explore' }">Explore view</router-link> to add intelligence
+            locally
+          </div>
+        </div>
       </div>
     </section>
 
+    <!-- Tag & label list -->
     <section class="section">
       <div class="container is-fluid">
-        <div v-if="this.meta.attributes.intelligence" class="card">
-          <header class="card-header">
-            <p class="card-header-title">External intelligence</p>
-          </header>
-          <div class="card-content">
-            <ts-dynamic-table
-              v-for="section in externalIntelligence.meta.sections"
-              v-bind:key="section.key"
-              :section="section"
-              :data="externalIntelligence.data[section.key]"
-            >
-            </ts-dynamic-table>
+        <div class="columns">
+          <!-- tag column -->
+          <div class="column">
+            <div class="box">
+              <h1 class="title">Tag List</h1>
+              <h1 class="subtitle">Tags that have been associated with IOCs.</h1>
+              <b-table v-if="Object.keys(tagInfo).length > 0" :data="Object.values(tagInfo)">
+                <b-table-column field="search" label="" v-slot="props" width="1em">
+                  <router-link :to="{ name: 'Explore', query: generateOrElasticQuery(props.row.iocs) }">
+                    <i class="fas fa-search" aria-hidden="true" title="Search sketch for all IOCs with this tag."></i>
+                  </router-link>
+                </b-table-column>
+                <b-table-column field="tagName" label="Tag name" v-slot="props" sortable>
+                  <b-tag type="is-info is-light">{{ props.row.tagName }} </b-tag>
+                </b-table-column>
+                <b-table-column field="count" label="IOCs tagged" v-slot="props" sortable numeric>
+                  {{ props.row.count }}
+                </b-table-column>
+              </b-table>
+              <span v-else>No IOCs have been tagged yet.</span>
+            </div>
+          </div>
+
+          <!-- labels column -->
+          <div class="column">
+            <div class="box">
+              <h1 class="title">Event tags</h1>
```

Comment:
Make this a bit smaller, maybe a h2

Reply:
Done - moved from boxes to cards, as changing the header type didn't change the font size.